code/ConvertToStandardPath_SubmarineCable.py

```diff
@@ -254,21 +225,40 @@ def get_data_from_database(db_file):
             for path in convert_multilinestring_to_list(cable_wkt):
                 cableIdToWkt[cable_id].add(str(path))
 
-    for cable_id, _, city_name, city_latitude, city_longitude in datas:
+    for cable_id, _, city_name, city_state, city_country, city_latitude, city_longitude in datas:
         if cable_id in wktToCities:
             wktToCities[cable_id].add(
```

Comment:
I think all the `wktToCities` are really `cableIdToCities`. Also by python naming conventions, it should be called `cable_id_to_cities`.

Reply:
fixed by cable_id_to_cities, also the cableIdToWkt has also been changed to cable_id_to_wkt